docs/tutorial/example-study/codelists/codelists.json

```diff
@@ -11,6 +11,12 @@
       "url": "https://www.opencodelists.org/codelist/opensafely/ethnicity/2020-04-27/",
       "downloaded_at": "2023-10-02 10:13:20.621822Z",
       "sha": "3d2f3998b71863a80b24d60f6f4282e14191ae13"
+    },
+    "bristol-asthma-exacerbations.csv": {
+      "id": "bristol/asthma-exacerbations/73c4eace",
+      "url": "https://www.opencodelists.org/codelist/bristol/asthma-exacerbations/73c4eace/",
```

Comment:
Yeah, I did wonder about that. It's certainly not as neat as having everything be from the OpenSAFLEY org. But I couldn't think of a specific problem it would create. We're linking to a fixed version so it's not like they can change it out from under us.